dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
Indeed I think it makes sense to have `coq_makefile` output a kind of "official" timing format, and then allow the tools to compare one or more timing logs easily.